### PR TITLE
Single shared config

### PR DESCRIPTION
**Pull request by cpaelzer**
Hi,
I wondered if we have had a case of proliferation with the configs when splitting the packages.
This is based on PR #18 and extends on it unifying the .conf files in (I hope) a better way.
We still would have the agent not run by default and the server having a config, but also the ability to deploy an own one.
At least I didn't see a drawback to what we have, but a massive win due to removing duplication and confusion.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -21,22 +21,24 @@ Build-Depends: debhelper (>= 11),
...
+         python3-keylime-tenant,
+         ${misc:Depends},
 Description: Keylime - full local install
  ${Description}
  .
- This is a metapackage meant for compatibility with former versions and will
- depend on python3-keylime-server and python3-keylime-agent as well as
- python3-keylime-agent-local bringing all services onto the local system
+ This is a metapackage meant for compatibility with former versions
+ and will depend on python3-keylime-tenant, python3-keylime-server,
```
Comment (mirespace) [inline debian/control:39]: We have here one tenant too many ;)

Comment (cpaelzer) [reply debian/control]: Thanks, fixed and pushed

**Comment by mirespace** [comment]
It seems to me the easiest way to handle the config problem until the split happens on upstream. Thanks a lot @cpaelzer !

File: debian/python3-keylime-server.install [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
 usr/bin/keylime_provider_platform_init
 usr/bin/keylime_provider_registrar
 usr/bin/keylime_provider_vtpm_add
 usr/bin/keylime_registrar
-usr/bin/keylime_tenant
 usr/bin/keylime_userdata_encrypt
+usr/bin/keylime_verifier
 usr/bin/keylime_webapp
-
-tpm_cert_store var/lib/keylime
```
Comment (utkarsh2102) [inline debian/python3-keylime-server.install:13]: When `/var/lib/keylime/tpm_cert_store` is moved to `debian/python3-keylime-tenant.install`, then having `var/lib/keylime` in `debian/python3-keylime-server.dirs` is redundant, I believe.

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
Thank you, @cpaelzer! Everything looks good except for:
```
utkarsh@tpm2:~/keylime/python-keylime$ sudo apt install ../python3-keylime-server_6.1.1-2~local1_all.deb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'python3-keylime-server' instead of '../python3-keylime-server_6.1.1-2~local1_all.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python3-keylime-server : Depends: python3-keylime-lib but it is not installable
                          Recommends: python3-keylime-defaultconf but it is not installable
                          Recommends: python3-keylime-doc but it is not installable
                          Recommends: python3-keylime-tenant but it is not installable
E: Unable to correct problems, you have held broken packages.
```

This looks odd, I am not sure what caused this but this is what I get when I try to install the -server. I'm taking a further look to understand what's happening.

File: debian/python3-keylime-server.install (hunk 2 above)
Comment (THS-on) [reply debian/python3-keylime-server.install:13]: We still need `var/lib/keylime` for the CA and the sqlite databases.

Comment (cpaelzer) [reply debian/python3-keylime-server.install:13]: Thanks @THS-on - so we can keep it in .dirs - Resolving this thread.

**Comment by utkarsh2102** [comment]
Hiya,

> This looks odd, I am not sure what caused this but this is what I get when I try to install the -server. I'm taking a further look to understand what's happening.

My bad. I've got everything working; all services seem to be up and running. And that's good! \o/

**Review by THS-on — APPROVED** [review]
@utkarsh2102 looks good to me :+1:

**Pull request merged**